Extensions/BBText/bbtextruntimeobject.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * An object displaying a BBCode formated text on the screen.
+ * @memberof gdjs
+ * @class BBTextRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.BBTextRuntimeObject = function(runtimeScene, objectData) {
+  gdjs.RuntimeObject.call(this, runtimeScene, objectData);
+
+  /** @type number */
+  this._opacity = objectData.content.opacity;
+  /** @type boolean */
+  this._visible = objectData.content.visible;
+  /** @type string */
+  this._text = objectData.content.text;
+  /** @type string */
+  this._color = objectData.content.color;
+  /** @type string */
+  this._family = objectData.content.family;
+  /** @type number */
+  this._size = objectData.content.size;
+  /** @type number */
+  this._wrappingWidth = objectData.content.width;
```

Comment:
objectData.content.width seems to be undefined - I don't see it in your object properties? 
If yes remove this and declare this._wrappingWidth to a sensible value. Either null or 0?